js/src/components/app-select-control/index.js

```diff
@@ -16,14 +17,54 @@ import './index.scss';
  * it will be added to the container div's `className`,
  * so that you can further control its style.
  *
- * @param {*} props
+ * @param {*} props The component props.
+ * @param {boolean} [props.autoSelectFirstOption=false] Whether the automatically select the first option.
```

Comment:
It would be nicer to indicate that when there is only one option, it will also change the UI presentation and will not allow interacting with the `select` control.

Reply:
👍